README.md

```diff
@@ -4,6 +4,11 @@ OCS CI is a framework to test Red Hat OCS features and functionality using AWS
 and other supported platforms. The framework is based on CephCI
   ( https://github.com/red-hat-storage/cephci )
 
+## Automatically generated documentation
+
+We have now pirmary documentation available at
+[readthedocs.io](https://ocs-ci.readthedocs.io/en/latest/)
```

Comment:
This link leads to a "Welcome to Read the Docs" page because apparently we do not have an `index.rst` or `README.rst` file under the root (or /docs) directory. However, I see that you have added an `index.rst` in this PR. Were these docs generated before adding this file or is this not working as expected?

This is what I'm seeing at that URL:
![Screen Shot 2019-11-20 at 1 45 55 PM](https://user-images.githubusercontent.com/1605721/69280942-39c16200-0b9c-11ea-8fd8-1b03030b98d7.png)

Reply:
It is because now the preview is on Petr's page (`ocs-ci-pbalogh.readthedocs.io`) not on the final address `ocs-ci.readthedocs.io`.

> This is how the generated documentation looks like. **Currently on my link but once merged I will setup one for ocs-ci directly** . https://ocs-ci-pbalogh.readthedocs.io/en/latest/#

Reply:
Yes, as this PR is not merged yet I couldn't set the readthedocs correctly for this repo cause the changes are not in the master ;)